robot/packages/utils.go

```diff
@@ -182,18 +182,18 @@ func unpackFile(ctx context.Context, fromFile, toDir string) error {
 			// Ex: tar -czf package.tar.gz ./bin/module.exe
 			parent := filepath.Dir(path)
 			if err := os.MkdirAll(parent, 0o700); err != nil {
-				return errors.Wrapf(err, "failed to create directory %q", parent)
```

Comment:
I saw this one using %q while adjacent ones were using %s, so I figured standardizing on quoted strings was preferable. Happy to undo.